.ci/jobs.yml

```diff
@@ -1,7 +1,24 @@
 JOB:
```

Comment:
Should we also add a note regarding also adding to `tasks/function_test_groups.js` here?